apps/climatemappedafrica/src/utils/fetchJson.js

```diff
@@ -1,5 +1,28 @@
+const cache = new Map();
```

Comment:
We may need to rethink a better caching strategy on the UI. I don't think this strategy is best. Maybe track in an issue?

Reply:
#1066